File: BehsaApp/BehsaApp.Web/Modules/Behsa/FinancialRows/FinancialRowsForm.cs
using Serenity.ComponentModel;
using System;

namespace BehsaApp.Behsa.Forms;

[FormScript("Behsa.FinancialRows")]
[BasedOnRow(typeof(FinancialRowsRow), CheckNames = true)]
public class FinancialRowsForm
{
    public DateTime ChangeDate { get; set; }
    public long Amount { get; set; }
}

File: BehsaApp/BehsaApp.Web/Modules/ServerTypes/Behsa/FinancialRowsForm.ts
import { DateEditor, StringEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface FinancialRowsForm {
    ChangeDate: DateEditor;
    Amount: StringEditor;
}

export class FinancialRowsForm extends PrefixedContext {
    static formKey = 'Behsa.FinancialRows';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!FinancialRowsForm.init)  {
            FinancialRowsForm.init = true;

            var w0 = DateEditor;
            var w1 = StringEditor;

            initFormType(FinancialRowsForm, [
                'ChangeDate', w0,
                'Amount', w1
            ]);
        }
    }
}